cycling/models.py

```diff
@@ -100,24 +110,26 @@ def current_to_log_current(current):
     return numpy.log(abs(current) + 1e-5)
 
 
-
-#def temperature_to_arrhenius(temp):
+# def temperature_to_arrhenius(temp):
 #   return numpy.exp(-1. / (temp + 273))
 
 def make_sign_grid():
     return numpy.array([1., -1.])
 
+
 def make_temperature_grid(min_t, max_t, n_samples, my_cell_ids):
     if n_samples < 2:
         n_samples = 2
 
-    my_files = CyclingFile.objects.filter(database_file__deprecated=False).filter(database_file__valid_metadata__cell_id__in=my_cell_ids)
-    my_max = my_files.aggregate(Max("database_file__valid_metadata__temperature"))[
-            "database_file__valid_metadata__temperature__max"
-        ]
-    my_min = my_files.aggregate(Min("database_file__valid_metadata__temperature"))[
-        "database_file__valid_metadata__temperature__min"
-    ]
+    my_files = CyclingFile.objects.filter(
+        database_file__deprecated = False
+    ).filter(database_file__valid_metadata__cell_id__in = my_cell_ids)
+    my_max = my_files.aggregate(
+        Max("database_file__valid_metadata__temperature")
+    )["database_file__valid_metadata__temperature__max"]
+    my_min = my_files.aggregate(
+        Min("database_file__valid_metadata__temperature")
+    )["database_file__valid_metadata__temperature__min"]
 
     my_max = clamp(min_t, my_max, max_t)
     if my_max < 55.:
```

Comment:
here we have some "magic numbers" would be good to add a TODO to add a parameter for these